db/migrate/20200513203600_create_moderator_applications.rb

```diff
@@ -0,0 +1,16 @@
+class CreateModeratorApplications < ActiveRecord::Migration[6.0]
+  def change
+    create_table :moderator_applications do |t|
```

Comment:
Really, the entire approach of using hard-coded columns for applications feels fundamentally flawed.